src/AMP.php

```diff
@@ -174,6 +174,17 @@ public function loadHtml($html, $options = [])
             $options['use_html5_parser'] = true;
         }
 
+        // By default the convertion of img into amp-anim is disabled (because of ressource cost)
```

Comment:
It might make sense to document this somewhere (we probably should add the config options to the readme).